scripts/installation/install-openkat.yml

```diff
@@ -0,0 +1,588 @@
+    postgresql_user:
+      name: rocky
+      password: "{{ rocky_db_password }}"
+      role_attr_flags: NOSUPERUSER
+
+  # sudo -u postgres createdb rocky_db
+  # sudo -u postgres psql -c 'GRANT ALL ON SCHEMA public TO rocky;' rocky_db
+  - name: Create Postgresql database rocky_db
+    become: true
+    become_user: postgres
+    become_method: sudo
+    postgresql_db:
+      name: rocky_db
+      owner: rocky
+      encoding: UTF-8
+      template: template0
+
+
+  # Now use rocky-cli to initialize the database:
+  # sudo -u kat rocky-cli migrate
+  - name: initialize Rocky database
+    become: true
+    become_user: kat
+    become_method: sudo
+    shell:
+      cmd: rocky-cli migrate
+
+  # sudo -u kat rocky-cli loaddata /usr/share/kat-rocky/OOI_database_seed.json
+  - name: Load data into OpenKAT
+    become: true
+    become_user: kat
+    become_method: sudo
+    shell:
+      cmd: rocky-cli loaddata /usr/share/kat-rocky/OOI_database_seed.json
+
+  # 
+  # The steps for creating the other databases will be similar, but we’ll explain them anyway for completeness.
+  # KAT-alogus DB
+  # 
+  # Generate a unique secure password for the KAT-alogus database user. You can use the same method we used for generating the Rocky database user password.
+  - name: Generate random password for KAT-alogus database user
+    set_fact:
+      kat_password: "{{ lookup('password', '/dev/null', length=20, chars=['ascii_letters'] ) }}"
+
+  - name: Show password for debugging purposes (TODO comment these lines)
+    debug:
+      var: kat_password
+
+  # Insert this password into the connection string for the KAT-alogus DB in /etc/kat/boefjes.conf. For example:
+  # KATALOGUS_DB_URI=postgresql://katalogus:<password>@localhost/katalogus_db
+  - name: Store password in /etc/kat/rocky.conf
+    lineinfile:
+      path: /etc/kat/boefjes.conf
+      regexp: "KATALOGUS_DB_URI=.*"
+      line:  "KATALOGUS_DB_URI=postgresql://katalogus:{{ kat_password }}@localhost/katalogus_db"
+
+  # Create a new database and user for KAT-alogus:
+  # 
+  # sudo -u postgres createuser katalogus -P
+  - name: Create Postgresql user katalogus and grant privs
+    become: true
+    become_user: postgres
+    become_method: sudo
+    postgresql_user:
+      name: katalogus
+      password: "{{ kat_password }}"
+      role_attr_flags: NOSUPERUSER
+
+  # sudo -u postgres createdb katalogus_db
+  # sudo -u postgres psql -c 'GRANT ALL ON SCHEMA public TO katalogus;' katalogus_db
+  - name: Create Postgresql database katalogus_db
+    become: true
+    become_user: postgres
+    become_method: sudo
+    postgresql_db:
+      name: katalogus_db
+      owner: katalogus
+      encoding: UTF-8
+      template: template0
+
+  # Initialize the database using the update-katalogus-db tool:
+  # 
+  # sudo -u kat update-katalogus-db
+  - name: initialize katalogus database
+    become: true
+    become_user: kat
+    become_method: sudo
+    shell:
+      cmd: update-katalogus-db
```

Comment:
Vermoedelijk een issue in het package, niet in ansible, maar ik krijg hierop een stacktrace:
kat@openkat:~$ update-katalogus-db 
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/opt/venvs/kat-boefjes/lib/python3.11/site-packages/alembic/__main__.py", line 4, in <module>
    main(prog="alembic")
  File "/opt/venvs/kat-boefjes/lib/python3.11/site-packages/alembic/config.py", line 630, in main
    CommandLine(prog=prog).main(argv=argv)
  File "/opt/venvs/kat-boefjes/lib/python3.11/site-packages/alembic/config.py", line 624, in main
    self.run_cmd(cfg, options)
  File "/opt/venvs/kat-boefjes/lib/python3.11/site-packages/alembic/config.py", line 601, in run_cmd
    fn(
  File "/opt/venvs/kat-boefjes/lib/python3.11/site-packages/alembic/command.py", line 385, in upgrade
    script.run_env()
  File "/opt/venvs/kat-boefjes/lib/python3.11/site-packages/alembic/script/base.py", line 578, in run_env
    util.load_python_file(self.dir, "env.py")
  File "/opt/venvs/kat-boefjes/lib/python3.11/site-packages/alembic/util/pyfiles.py", line 93, in load_python_file
    module = load_module_py(module_id, path)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/venvs/kat-boefjes/lib/python3.11/site-packages/alembic/util/pyfiles.py", line 109, in load_module_py
    spec.loader.exec_module(module)  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/venvs/kat-boefjes/lib/python3.11/site-packages/boefjes/migrations/env.py", line 6, in <module>
    from boefjes.config import settings
  File "/opt/venvs/kat-boefjes/lib/python3.11/site-packages/boefjes/config.py", line 94, in <module>
    settings = Settings()
               ^^^^^^^^^^
  File "pydantic/env_settings.py", line 40, in pydantic.env_settings.BaseSettings.__init__
    super().__init__(
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
    raise validation_error
pydantic.error_wrappers.ValidationError: 1 validation error for Settings
queue_uri
  ensure this value has at least 1 characters (type=value_error.any_str.min_length; limit_value=1)